micro-rdk/src/common/grpc.rs

```diff
@@ -35,7 +35,7 @@ use thiserror::Error;
 use super::webrtc::grpc::WebRtcGrpcService;
 
 #[cfg(feature = "camera")]
-static GRPC_BUFFER_SIZE: usize = 10240;
+static GRPC_BUFFER_SIZE: usize = 1024 * 1024; // 1MB
```

Comment:
grpc spec has an incoming message limit of 4MB. the current `fake_image.jpeg` is about 10KB, the rdk's `FakeCamera` saves a jpeg of ~20KB.